data-raw/presidential_speech/presidential_speech/spiders/inaug_text.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from random import randint
+
+
+class InaugTextSpider(scrapy.Spider):
+    name = "inaug_text"
+    allowed_domains = ["http://www.presidency.ucsb.edu/"]
+    start_urls = (
```

Comment:
Add a note explaining what these URLs are and how to keep this list up to date (if possible)